### PR TITLE
Remove old fe meta version

### PR DESCRIPTION
**Pull request by yiguolei**
# Proposed changes

Issue Number: close #xxx

## Problem Summary:

Describe the overview of changes.

## Checklist(Required)

1. Does it affect the original behavior: (Yes/No/I Don't know)
2. Has unit tests been added: (Yes/No/No Need)
3. Has document been added or modified: (Yes/No/No Need)
4. Does it need to update dependencies: (Yes/No)
5. Are there any changes that cannot be rolled back: (Yes/No)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/load/loadv2/LoadJob.java [hunk 1]
```diff
@@ -1030,10 +1029,6 @@ public void write(DataOutput out) throws IOException {
     }
 
     public void readFields(DataInput in) throws IOException {
-        if (Catalog.getCurrentCatalogJournalVersion() < FeMetaVersion.VERSION_94) {
-            readFieldOld(in);
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/load/loadv2/LoadJob.java:1034]: remove `readFieldOld` method

File: fe/fe-core/src/main/java/org/apache/doris/mysql/privilege/UserProperty.java [hunk 2]
```diff
@@ -523,86 +520,31 @@ public void write(DataOutput out) throws IOException {
...
-            in.readBoolean();
-
-            if (Catalog.getCurrentCatalogJournalVersion() >= FeMetaVersion.VERSION_1) {
-                // boolean isSuperuser
-                in.readBoolean();
-            }
-        }
+        qualifiedUser = Text.readString(in);
 
         if (Catalog.getCurrentCatalogJournalVersion() < FeMetaVersion.VERSION_100) {
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/mysql/privilege/UserProperty.java:525]: MINIMUM_VERSION_REQUIRED is 100, so I think we can remove this.

File: fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java [hunk 3]
```diff
@@ -975,35 +973,19 @@ private void readFromStream(DataInput in) throws IOException {
...
+        collationServer = Text.readString(in);
+        batchSize = in.readInt();
+        disableStreamPreaggregations = in.readBoolean();
+        parallelExecInstanceNum = in.readInt();
+        exchangeInstanceParallel = in.readInt();
     }
 
     public void readFields(DataInput in) throws IOException {
-        if (Catalog.getCurrentCatalogJournalVersion() < FeMetaVersion.VERSION_67) {
-            readFromStream(in);
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java:1003]: remove `readFromStream()` method

File: fe/fe-core/src/main/java/org/apache/doris/mysql/privilege/UserProperty.java (hunk 2 above)
Comment (yiguolei) [reply fe/fe-core/src/main/java/org/apache/doris/mysql/privilege/UserProperty.java:525]: Actually, I am removing code that FeMetaVersion < VERSION_96. I am afraid of version 100 is too latest.

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**